### PR TITLE
fixes issue #128, long text not wrapping on /brigades or /projects

### PR DESCRIPTION
**Pull request by rkaufman13**
Hi, this CSS fix should fix text wrapping on both `/brigades` and `/projects` as described in #128 

With all text wrapping, react-table defaults to table columns being equally wide, and the only way to set column widths appears to be using hardcoded pixel values (no percentages or flex-basis). That didn't look very good, so I also added a selector to target the "description" column and make it 40% of the table, which makes the other 3 columns share the remaining 60% equally. All this to say I'm totally happy if you wish to change the width of the description column to larger or smaller, but 40% looked fairly pleasing to my eye on a variety of monitor sizes.

### CODE REVIEW

File: statusboard/src/components/ProjectsTable/ProjectsTable.scss [hunk 1]
```diff
@@ -16,11 +17,12 @@
     width: 100%;
     border-spacing: 0;
     border-radius: inherit;
+    table-layout:fixed;
```
Comment (k3KAW8Pnf7mkmdSMPHz27) [inline statusboard/src/components/ProjectsTable/ProjectsTable.scss:20]: I didn't know about `table-layout: fixed`, but it seems good for this purpose! Even more so if we end up adding better paging/infinite scrolling.

**Review by k3KAW8Pnf7mkmdSMPHz27 — COMMENTED** [review]
> and the only way to set column widths appears to be using hardcoded pixel values (no percentages or flex-basis)

I am not sure I understand. Do you mean setting it from TS/JS instead of CSS? I think this is good, and it is reasonable to set it in `ProjectsTable.scss`.

I just have a few comments/questions, but otherwise, I'd go ahead and merge.

@mmazanec22 would you have time to take a look at this?

**Comment by rkaufman13** [comment]
Yes, we can hard-code column widths as a prop passed to each react-table
column, but the only values react-table accepts are pixels, rather than
percentages, so it wouldn't help in this case where we want to design for
multiple screen sizes :)
Table-layout:fixed is, I believe, required to set column widths that do not
resize based on the content, but I could be mistaken!

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
> Yes, we can hard-code column widths as a prop passed to each react-table
column, but the only values react-table accepts are pixels, rather than
percentages, so it wouldn't help in this case where we want to design for
multiple screen sizes :)

That is unfortunate :/

> Table-layout:fixed is, I believe, required to set column widths that do not
resize based on the content, but I could be mistaken!

Even if we didn't need to set the column widths, it seems to be the right table layout. So I am definitively pro this change :)

The only thing I am not sure about is the change in margin for the header row,

**https://projects.brigade.network/**
Skärmavbild 2021-09-08 kl  17 51 58

**[Preview](https://deploy-preview-130--dreamy-snyder-8339ae.netlify.app/)**
Skärmavbild 2021-09-08 kl  17 52 33

I like the extra space but it behaves slightly worse on small screens, so I'd be ok with keeping this change, I just want to make sure it is intentional. That is why I asked about the `box-margin` earlier. Except for that, this looks good to me and is good to go!

**Comment by rkaufman13** [comment]
Yes, so...it's intentional, but it's a tradeoff. I think the choice is
between headers being slightly less readable on small screens or the table
being slightly less readable on small screens. I personally think this is
the lesser of two evils but leave it up toyou :)

**Review by k3KAW8Pnf7mkmdSMPHz27 — APPROVED** [review]
> I think the choice is
between headers being slightly less readable on small screens or the table
being slightly less readable on small screens. I personally think this is
the lesser of two evils but leave it up to you :)

Ok. Well, in that case, this is good to go! Thank you for you contributions and sorry about the delay!

**Pull request merged**